docs/UserGuide.md

```diff
@@ -156,7 +156,7 @@ Examples:
 
 ### Finding a person: `find`
 
-Format: ‘find n/NAME p/PHONE_NUMBER r/ROOM_NUMBER t/Tags’’
+Format: ‘find n/NAME p/PHONE_NUMBER r/ROOM_NUMBER t/TAG’
 
 * any possible orders and combinations of the 3 parameters name, phone number and room number are applicable
```

Comment:
Should these be capital letters and follow the NAME, PHONE_NUMBER, ROOM_NUMBER format, or would it be better to keep it like this?